packages/integration/package.json

```diff
@@ -34,7 +34,7 @@
   "dependencies": {
     "@backstage/config": "workspace:^",
     "@backstage/errors": "workspace:^",
-    "@octokit/auth-app": "^4.0.0",
+    "@octokit/auth-app": "4.0.9",
```

Comment:
This needs a changeset. But can we revert these? Are the changes really necessary? You didn't provide an explanation for what you're achieving.

If this is about the same thing that [these resolutions](https://github.com/backstage/backstage/blob/667c490cc10fdb6dea06ccfe7ef2ab92364d7d45/packages/create-app/templates/default-app/package.json.hbs#L40) address, then you should probably revert these bumps and use resolutions for now. The octokit types are just broken temporarily and we're waiting for them to ship a fix.